aea/helpers/ipfs/base.py

```diff
@@ -92,8 +101,63 @@ def get(self, file_path: str) -> str:
         """
         file_b = _read(file_path)
         file_pb = self._pb_serialize_file(file_b)
-        ipfs_hash = self._generate_multihash(file_pb)
-        return ipfs_hash
+        return self._generate_multihash(file_pb)
+
+    def hash_directory(self, dir_path: str) -> str:
```

Comment:
Looks legit - can you point in docstring to the code from which this is borrowed on the IPFS golang side?

Reply:
I didn't pick anything from go-ipfs, I watched a YouTube video on IPFS file hashing and content addressing, From there I got some idea on hashing and I got some intuition from Yuri's code.